test/params/str-list.spec.js

```diff
@@ -6,6 +6,22 @@ describe('Testing StrList Parameter', () => {
   const jsonParam = api.StrList('list', 'json');
   const queryParamEnum = api.StrList('list', 'query', { enums: ['enumOne', 'enumTwo', 'enumThree'] });
   const bodyParamEnum = api.StrList('list', 'json', { enums: ['enumOne', 'enumTwo', 'enumThree'] });
+  const queryParamInValidConstraints = api.StrList('list', 'query', {
```

Comment:
In Valid reads too close to Invalid, makes it hard to read the tests :)
I thought you were testing invalid constraints below, I'd rename this.

Constraints is also a very general term. If we get our coverage through the unit tests, we're okay.